### ISSUE TITLE
New whatsapp tracking domain

### ISSUE THREAD

**Issue by lthedetective** [hagezi/dns-blocklists#1868]
### Which domain(s) should be blocked?

graph.whatsapp.com

### Why should these domain(s) be blocked?

trackers used by other meta services
graph.instagram.com
graph.facebook.com

**Comment by hagezi** [comment]
Please provide more details, e.g. what triggers the domain call?

**Comment by lthedetective** [comment]
Every time you open whatsapp.
I have blocked it in my nextdns profile for almost a month now and haven't faced much issues.

**Comment by lthedetective** [comment]
![Screenshot_20231126_183107](https://github.com/hagezi/dns-blocklists/assets/122800876/15f894ab-c11b-4bf8-a384-0b235bf89310)

**Comment by hagezi** [comment]
On iOS, the domain is not triggered by the app. I have searched the DNS logs of the family networks and have only found one call so far, on an Android device.

**Comment by hagezi** [comment]
On Whatsapp web I see calls to `graph.whatsapp.net` but not to `graph.whatsapp.com`

**Comment by lthedetective** [comment]
Even in my family that domain calls are not that common compared to graph.facebook.com but still it has multiple calls every day. Also It might be region based.

**Comment by lthedetective** [comment]
The calls happens after about 5 mins and after clearing the app from the ram.

**Comment by hagezi** [comment]
Yes, I've already thought about the fact that it depends on the region. I have now blocked `graph.whatsapp.net|com` in the networks. Let's see ...

**Comment by hagezi** [comment]
I have not found any restrictions so far, I will block the domains for testing in the Ultimate.

**Comment by yoshimo** [comment]
So far it seems the ```web.whatsapp.com```uses the ```.net``` version on a laptop and the mobile android devices go for ```graph.whatsapp.com```

**Comment by hagezi** [comment]
Yes, I can also see that in the logs. There have been no calls on iOS so far.

**Comment by lthedetective** [comment]
In android in my country the domain calls have become very common every time I open the app after closing it from the ram.

**Comment by user334** [comment]
Hello @yoshimo 
> Data send to `graph.whatsapp.net/wa_qpl_data/`

Where did you get this exact address from and have you observed WhatsApp to _actually_ send any data there?

**Comment by yoshimo** [comment]
Browser developer tools when you use the web.whatsapp.com client on your computer

**Comment by hagezi** [comment]
FYI: The blocking of the graph domains breaks the whatsapp avatar creation.

**Comment by hagezi** [comment]
@AleIlMagno I will block the domains again in the Ultimate in the next release. They should actually only be unblocked in the lists above the Ultimate.

**Comment by hagezi** [comment]
Thanks for the tip. I had unblocked them this morning, but wasn't quite awake yet ;)
